extensions/cache/deployment/src/main/java/io/quarkus/cache/deployment/CacheConfig.java

```diff
@@ -0,0 +1,45 @@
+package io.quarkus.cache.deployment;
+
+import java.time.Duration;
+import java.util.Map;
+import java.util.Optional;
+import java.util.OptionalLong;
+
+import io.quarkus.runtime.annotations.ConfigGroup;
+import io.quarkus.runtime.annotations.ConfigItem;
+import io.quarkus.runtime.annotations.ConfigRoot;
+
+/**
+ * @author Gwenneg Lepage
+ */
+@ConfigRoot
+public class CacheConfig {
```

Comment:
More configuration options will probably be added soon, this is just a first (untested for now) version.